mbedtls/Cargo.toml

```diff
@@ -30,7 +30,7 @@ num-bigint = { version = "0.2", optional = true }
 bit-vec = { version = "0.5", optional = true }
 block-modes = { version = "0.3", optional = true }
 rc2 = { version = "0.3", optional = true }
-tokio = { version = "1.26.0", optional = true }
+tokio = { version = "1.16.1", optional = true }
```

Comment:
Why the downgrade?

Reply:
It's because the `mio` lib used by `tokio` with version > 1.16.1 depends on `libc` >= 0.2.121 but "2.28.0" `mbedtls-sys-auto` need `libc` <= 0.2.81

Reply:
So `tokio` with version = "1.16.1" is the highest one we could use if we do not update dependencies in  "2.28.0" `mbedtls-sys-auto`

Reply:
Right, and this is not even a downgrade w.r.t. the target branch. Thanks.